WTWSMS/history/provinces/478 - Monemvasia.txt

```diff
@@ -4,13 +4,13 @@
 title = c_monemvasia
 
 # Settlements
-max_settlements = 4
+max_settlements = 6
```

Comment:
I wasn't talking about prosperity. I was talking about empty slots for potential holdings.